app/mailers/requests_confirmation_mailer.rb

```diff
@@ -10,21 +10,17 @@ def confirmation_email(request)
 
   private
 
+  # fetch_items
+  #
+  # Fetches sorted items associated with the current Request object.
+  #
+  # @return Array of request_items with item names
   def fetch_items(request)
```

Comment:
@felipelovato @sean-dickinson soooo this change is going to conflict with what y'all are doing right now. It'll be a race to see which we merge first :)

Reply:
OK -- please undo this whole refactor; it is going to conflict with other units work and also I think a future refactor can eliminate this altogether (switching to direct usage of `request.item_requests` both here and in the email templates).

Reply:
Done. I just kept the TODO line to do the refactor eventually.